.github/workflows/_buildpacks-release.yml

```diff
@@ -90,6 +90,44 @@ jobs:
         id: generate-changelog
         run: actions generate-changelog --version ${{ steps.generate-buildpack-matrix.outputs.version }}
 
+      - name: Temporary fix for bash-based buildpacks
+        run: |
+          buildpacks='${{ steps.generate-buildpack-matrix.outputs.buildpacks }}'
+
+          bash_buildpack_source_dirs=()
+          bash_buildpack_output_dirs=()
+
+          # copy any bash-based buildpack to target buildpack dir because `cargo libcnb package` will ignore them
+          for buildpack in $(jq -c '.[]' <<< "${buildpacks}"); do
+            buildpack_dir=$(jq -r '.buildpack_dir' <<< "${buildpack}")
+            output_dir=$(jq -r '.buildpack_output_dir' <<< "${buildpack}")
+            if [ ! -d "${output_dir}" ]; then
+              echo "bash-based buildpack detected at ${buildpack_dir}"
+              cp -R "${buildpack_dir}" "${output_dir}"
+              bash_buildpack_source_dirs+=("${buildpack_dir}")
+              bash_buildpack_output_dirs+=("${output_dir}")
+            fi
+          done
+
+          # replace dependencies that reference a bash-buildpack
+          for buildpack in $(jq -c '.[]' <<< "${buildpacks}"); do
+            output_dir=$(jq -r '.buildpack_output_dir' <<< "${buildpack}")
+            echo "checking dependencies in ${output_dir}/package.toml"
+            for dep in $(yq -oy '.dependencies[].uri' "${output_dir}/package.toml"); do
```

Comment:
Not all buildpacks have a `package.toml` (since the file is optional if a buildpack has no deps) - does this need to be wrapped in a conditional checking if the file exists? (or a `continue` perhaps, to reduce the level of nesting?)

Reply:
the `cargo libcnb package` output always produces a `package.toml` even if one isn't present in the buildpack source directory.  the default version being:

```toml
[buildpack]
uri = "."
```

this is so we always have a config file present to use when running `pack`.  

the only way this could error is if we copy over a bash-based buildpack that doesn't have a `package.toml` because we loop over the buildpack directories again. i could guard against that case but i also happen to know there is one in the bash-based npm buildpack.